docs/table/access_table.rst

```diff
@@ -374,6 +374,49 @@ For columns the syntax and behavior of
   2973.0
   Length = 100 rows
 
+  Column alignment
+''''''''''''''''
```

Comment:
@bsipocz Ah, awesome! Thank you so much. I also have a very difficult time finding these kinds of errors :)